From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Wed, 2 Feb 2011 18:31:52 +0100
Subject: [PATCH] add missing subdir

"widgets" is missing from SUBDIRS and wont be commpiled.

This patch addes it.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 examples/tutorials/tutorials.pro |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/examples/tutorials/tutorials.pro b/examples/tutorials/tutorials.pro
index 949fdf6..89febd0 100644
--- a/examples/tutorials/tutorials.pro
+++ b/examples/tutorials/tutorials.pro
@@ -1,5 +1,6 @@
 TEMPLATE      = subdirs
 SUBDIRS       = \
+                widgets \
                 addressbook
 
 # install
-- 
1.7.2.3

